### PR TITLE
Convert kubernetes-e2e-upgrades-image-gke 1.4-->latest jobs

### PR DESCRIPTION
**Pull request by xiangpengzhao**
#1230 #1866 #1871 #1881 #1882 #1884 

This PR should be merged after #1884 and then rebased. gke upgrade jobs are finished finally...

@krzyzacy

### CODE REVIEW

**Comment by krzyzacy** [comment]
@k8s-bot ok to test

**Comment by krzyzacy** [comment]
@xiangpengzhao sorry backlogged, that's rocket speed :-)

**Comment by xiangpengzhao** [comment]
Rebased. Bazel test should be successful after #1882 and #1884 merged. Seems not so good a way to create PRs like this...
@krzyzacy

**Comment by krzyzacy** [comment]
@xiangpengzhao you should include your previous commits here, so that there would be no test failures, and once rebased it will be automatically resolved.

For example, from branch 1882 you can create a branch 1883 with 1882's commits and work on it :-)

File: jobs/ci-kubernetes-e2e-gke-container_vm-1.4-container_vm-latest-upgrade-cluster-new.env [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### job-env
+
+E2E_OPT=--check_version_skew=false
+E2E_UPGRADE_TEST=true
+GINKGO_UPGRADE_TEST_ARGS=--ginkgo.focus=\[Feature:ClusterUpgrade\] --upgrade-target=ci/latest-latest --upgrade-image=container_vm
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest-latest
```
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-gke-container_vm-1.4-container_vm-latest-upgrade-cluster-new.env]: here as well :-) just point out

**Comment by xiangpengzhao** [comment]
Sounds great! Will work that way in future:-)

**Comment by krzyzacy** [comment]
last one, yay

**Comment by xiangpengzhao** [comment]
Rebased. :v: :smile: :v:

**Comment by krzyzacy** [comment]
/lgtm

woooo how do you type emoji here

**Comment by krzyzacy** [comment]
only 13.9% shell in test-infra :1st_place_medal:

**Comment by xiangpengzhao** [comment]
You've got it! :+1: 
I would like to go on converting the left jobs. But these seem like a little complicated. I'm trying to understand. Any suggestion on what to do next? @krzyzacy

**Comment by xiangpengzhao** [comment]
Created a new PR #1899 to remove files of lingering 1.2 skew jobs. Then I will try to convert soak jobs according to your PR #1829.
@krzyzacy Sen, is that OK?

**Comment by krzyzacy** [comment]
@xiangpengzhao given https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-soak-gce-1.3-test/644 looks pretty green, I think it should be safe now. Feel free to send a PR and I'll take a look.

**Comment by xiangpengzhao** [comment]
OK. Will do it :-)

**Pull request merged**